felix/bpf-apache/bpf.h

```diff
@@ -16,17 +16,19 @@
 #define __CALI_BPF_H__
 
 #include <linux/bpf.h>
+#include <bpf/bpf_helpers.h>
+
 #include <stddef.h>
 #include <linux/ip.h>
 
 /* Kernel/libbpf bpf_helpers.h also contain this struct 'bpf_map_def' */
-struct bpf_map_def {
+/*struct bpf_map_def {
         unsigned int type;
         unsigned int key_size;
         unsigned int value_size;
         unsigned int max_entries;
         unsigned int map_flags;
-};
```

Comment:
this should be removed completely, right?

Reply:
yeap. removed. now.